platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsManageDocumentsInSpaceTestIT.java

```diff
@@ -1,102 +1,139 @@
+        spaceSettingManagement = new SpaceSettingManagement(this);
+        homePagePlatform = new HomePagePlatform(this);
+        documentManagement = new DocumentManagement(this);
+
+    }
+
+    @Tag("ECMS-7631")
+    @Test
+    public void test01_CheckSaveASNewVersionButtonWhenUserNotHaveModifyRight() {
+        String space = "space" + getRandomNumber();
+        info("Create a space");
+        homePagePlatform.goToMySpaces();
+        spaceManagement.addNewSpaceSimple(space, space);
+        spaceHomePage.goToSpaceSettingTab();
+        spaceSettingManagement.goToMemberTabInSpaceSettingTab();
+        ELEMENT_INPUT_INVITE_USER.sendKeys(PLFData.DATA_USER2);
+        $(ELEMENT_SPACE_BTN_INVITE).click();
+        manageLogInOut.signIn(PLFData.DATA_USER2, password);
+        homePagePlatform.goToMySpaces();
+        spaceManagement.acceptAInvitation(space);
+        manageLogInOut.signIn(PLFData.DATA_USER1, "gtngtn");
+        homePagePlatform.goToMySpaces();
+        ELEMENT_SPACES_LIST.find(byText(space)).click();
+        spaceManagement.goToDocumentTab();
+        $(byId("MultiUploadInputFiles")).uploadFromClasspath("eXo-Platform.png");
+        ELEMENT_LIST_DOCUMENTS_IN_SPACE.find(byText("eXo-Platform.png")).waitUntil(Condition.visible, Configuration.timeout).click();
+        documentManagement.goToPermissions();
+        $(byId(ELEMENT_CHECKBOX_MODIFY_PERMISSION.replace("{user}", space))).parent().waitWhile(Condition.not(Condition.visible),
+                Configuration.timeout);
+        if ($(byId(ELEMENT_CHECKBOX_MODIFY_PERMISSION.replace("{user}", space))).is(Condition.checked))
+            $(byId(ELEMENT_CHECKBOX_MODIFY_PERMISSION.replace("{user}", space))).parent().click();
+        $(ELEMENT_CLOSE_BTN).click();
+        manageLogInOut.signIn(PLFData.DATA_USER2, password);
+        homePagePlatform.goToMySpaces();
+        ELEMENT_SPACES_LIST.find(byText(space)).click();
+        spaceManagement.goToDocumentTab();
+        ELEMENT_LIST_DOCUMENTS_IN_SPACE.find(byText("eXo-Platform.png"))
+                .waitUntil(Condition.visible, Configuration.timeout)
+                .contextClick();
+        ELEMENT_BUTTON_UPLOAD_NEW_VERSION.shouldNotBe(Condition.visible);
+        ELEMENT_BUTTON_CREATE_NEW_VERSION.shouldNotBe(Condition.visible);
+        ELEMENT_LIST_DOCUMENTS_IN_SPACE.find(byText("eXo-Platform.png")).waitUntil(Condition.visible, Configuration.timeout).click();
+        ELEMENT_BUTTON_UPLOAD_NEW_VERSION.shouldNotBe(Condition.visible);
+        ELEMENT_BUTTON_CREATE_NEW_VERSION.shouldNotBe(Condition.visible);
+        manageLogInOut.signIn(PLFData.DATA_USER1, "gtngtn");
+        homePagePlatform.goToMySpaces();
+        spaceManagement.deleteSpace(space, false);
+
+    }
+
+    @Test
+    public void test02_CheckScriptNotExecutedWhenFilePathContainsScript() {
+        //8357
+        String space = "space" + getRandomNumber();
+        String folderTitle = "test<script>alert(2);</alert>";
```

Comment:
Should be "test<script>alert(2);</script>" no ?

Reply:
So please change it to "test<script>alert(2);</script>".

Reply:
Ok done.